tests/chainer_tests/test_link.py

```diff
@@ -1155,13 +1165,21 @@ def test_copy_with_copy_mode(self):
         self.l2.x.initializer = initializers.Normal(
             dtype=self.l2.x.initializer.dtype)
         self.l2.x.initialize(self.l2.x.shape)
+        self.x.initializer = initializers.Normal(
+            dtype=self.x.initializer.dtype)
+        self.x.initialize(self.x.shape)
 
         c2 = self.c2.copy(mode='copy')
         self.assertIs(c2.name, None)
         self.assertIsInstance(c2._children, set)
+        self.assertTrue(hasattr(c2, 'x'))
+        self.assertIsNot(c2.x, self.x)
+        self.assertIsNot(c2.x.data, self.x.data)
```

Comment:
How about this check: `self.assertTrue(numpy.array_equal(c2.x.data, self.x.data))`

Reply:
Should we change the other comparissons?

It is checking the Link parameters using the same `assertIs/assertIsNot`
I am fine with changing all of them.

Reply:
No, I mean adding this check just like other parameters like `c2.c1.l1.x` below.

Reply:
Ah, ok! I missed that one 🙂